core/src/main/java/pl/agh/edu/actor/component/button/ScenarioButton.java

```diff
@@ -0,0 +1,74 @@
+package pl.agh.edu.actor.component.button;
+
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.scenes.scene2d.ui.*;
+import com.badlogic.gdx.scenes.scene2d.utils.NinePatchDrawable;
+
+import pl.agh.edu.actor.GameSkin;
+import pl.agh.edu.config.GraphicConfig;
+import pl.agh.edu.enums.HotelType;
+
+public class ScenarioButton extends Table {
+	private final Skin skin = GameSkin.getInstance();
+	public final HotelType hotelType;
+	private boolean isSelected = false;
+	public int width = GraphicConfig.getResolution().WIDTH;
+	public int height = GraphicConfig.getResolution().HEIGHT;
+
+	public ScenarioButton(String title, String image, String description, String season, HotelType hotelType, String titleFont, String textFont) {
+		super();
+		this.hotelType = hotelType;
+		pad(30, 20, 30, 20);
```

Comment:
wsyzstkie te wartości ustawił na objet towrayszący które zwraca nam w metodkach aby póżniej było to wygodnie modyfikować